llama_stack/providers/inline/safety/prompt_guard/prompt_guard.py

```diff
@@ -114,8 +116,10 @@ async def run(self, messages: list[Message]) -> RunShieldResponse:
         elif self.config.guard_type == PromptGuardType.jailbreak.value and score_malicious > self.threshold:
             violation = SafetyViolation(
                 violation_level=ViolationLevel.ERROR,
-                violation_type=f"prompt_injection:malicious={score_malicious}",
-                violation_return_message="Sorry, I cannot do this.",
+                user_message="Sorry, I cannot do this.",
```

Comment:
Why the rename from `violation_return_message`?

Reply:
good point, so we changed  `ShieldResponse` to most recent `SafetyViolation`, and used the `user_message` according to `Consistency with modern API design`